packages/next/pages/_document.tsx

```diff
@@ -579,6 +607,22 @@ export class NextScript extends Component<OriginProps> {
     })
   }
 
+  getScriptLoaderScripts() {
```

Comment:
Nit `getBeforeNextJSScripts` or something conveying that only the files which are really required before hydration